.gitignore

```diff
@@ -348,3 +348,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+#vscode launch and tasks json
+.vscode/
```

Comment:
💡 Shall we remove the `.vscode` dir structure from this repo? could be in separate PR since we are gitignoring it.

Reply:
.vscode directory doesn't exist just added them to the .gitignore and directory is removed. are you saying we want this in separate PR ?

Reply:
💡☕️ Cool I meant this .vscode dir which currently exist in codebase: https://github.com/Azure/Bridge-To-Kubernetes/tree/main/src/.vscode , son of we are gitignoring then just bringing this dot in your attention. 😊

Reply:
Thank you @Tatsinnit missed checking in the main repo, I will remove them.